action.yml

```
@@ -1,5 +1,5 @@
-name: 'Wolfia magic link'
-description: 'Generate a wolfia magic link and create a comment on the PR with a magic link'
+name: 'Wolfia deploy'
+description: 'Upload your app to automating your mobile app releases on [Wolfia](https://wolfia.com)'
```

```suggestion
description: 'Upload your app to automate your mobile app releases on [Wolfia](https://wolfia.com)'
```

`Upload your [binary]`?

---

This will be the description on GitHub marketplace, so keeping it generic